tests/pages/settings/profile-page.ts

```diff
@@ -1,6 +1,11 @@
-import { create, triggerable, visitable } from 'ember-cli-page-object';
+import { clickable, triggerable, visitable } from 'ember-cli-page-object';
+import createPage from 'codecrafters-frontend/tests/support/create-page';
+
+export default createPage({
+  anonymousModeToggle: {
+    toggle: clickable('[data-test-anonymous-mode-toggle]', { resetScope: true }),
```

Comment:
Not sure why we need `resetScope` here - does it work without it? `resetScope` is usually only required if you're dealing with things like dropdown where the div might get placed outside a component.